lib/gui/app/app.js

```diff
@@ -224,7 +224,7 @@ app.run(($timeout) => {
     $timeout(() => {
       if (BLACKLISTED_DRIVES.length) {
         const allowedDrives = drives.filter((drive) => {
-          return !BLACKLISTED_DRIVES.includes(drive.device)
+          return !BLACKLISTED_DRIVES.includes(drive.devicePath)
```

Comment:
I think it'd be worth allowing to blacklist based on both, the `.device` and the `.devicePath`, i.e.
```js
return !(
  BLACKLISTED_DRIVES.includes(drive.devicePath) ||
  BLACKLISTED_DRIVES.includes(drive.device)
)
```

Reply:
...or perhaps there could be separate `BLACKLISTED_DRIVES` and `BLACKLISTED_DEVICES` env-vars?